docs/QUnit/test.each.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: QUnit.test.each()
+excerpt: Add a parameterized test to run.
+categories:
+  - main
+  - async
+version_added: "1.0"
```

Comment:
Set this to `"unreleased"` for now.

Reply:
Done. Is this going to be updated automatically with a new release?

Reply:
Not at the moment. I try to remember them, or grep for it, during a release and fill them in as part of the release prep commit. While our release cadence is increasing through past and this year, I don't expect features to accumulate. So, for now, my view of the [Is It Worth The Time?](https://xkcd.com/1205/) table says that it's not worth automating.

Having said that, it seems like a simple thing to automate and so I wouldn't be worried about it complicating our maintenance work. If you feel compelled to give it a try (perhaps as a way to familiarize with the release process), it would likely take the form of an additional step in [build/prep-release.js](https://github.com/qunitjs/qunit/blob/2.15.0/build/prep-release.js).

Reply:
Thanks. Let me look into that after https://github.com/qunitjs/qunit/pull/1614.